lib/suma/admin_api/members.rb

```diff
@@ -188,5 +188,7 @@ class DetailedMemberEntity < MemberEntity
     expose :sessions, with: MemberSessionEntity
     expose :orders, with: MemberOrderEntity
     expose :message_deliveries, with: MessageDeliveryEntity
+    expose :preferences, with: MessagePreferenceSubscriptionEntity,
```

Comment:
I don't think this is ideal (should be like `{preferences: {subscriptions: []}}` but it's admin so easy to change later.